Remotion/ObjectBinding/Web.UnitTests/UI/Controls/Validation/BocListValidationResultDispatchingValidatorTest.cs

```diff
@@ -149,8 +205,21 @@ public void RefreshErrorMessage_WithValidationFailuresForBoundPropertyAndRowFail
                                              };
 
       var validationFailureRepoStub = new Mock<IBocListValidationFailureRepository>();
-      validationFailureRepoStub.Setup(_ => _.GetUnhandledValidationFailuresForBocList(true)).Returns(validationFailuresForBocList);
-      validationFailureRepoStub.Setup(_ => _.GetUnhandledValidationFailuresForBocListAndContainingDataRowsAndDataCells(true)).Returns(validationFailuresForBocListRows);
+      validationFailureRepoStub
+          .Setup(_ => _.GetUnhandledValidationFailuresForBocList(true))
+          .Returns(validationFailuresForBocList);
+
+      var sequence = new MockSequence();
+      validationFailureRepoStub
+          .InSequence(sequence)
+          .Setup(_ => _.GetUnhandledValidationFailuresForBocListAndContainingDataRowsAndDataCells(true))
+          .Returns(validationFailuresForBocListRows);
+      validationFailureRepoStub
+          .InSequence(sequence)
+          .Setup(_ => _.GetUnhandledValidationFailuresForBocListAndContainingDataRowsAndDataCells(true))
+          .Returns(Array.Empty<BocListValidationFailureWithLocationInformation>());
+
```

Comment:
Please change to use SetupSequence and multiple returns isntead. For ease of maintenance, I want to avoid `MockSquence` in the codebase.